lab_4/main.py

```diff
@@ -4,44 +4,141 @@
 
 from ngrams.ngram_trie import NGramTrie
 
+SENTENCE_END = "<END>"
```

Comment:
If it is used only in `tokenize_by_sentence`, the better approach is to place it there.

Reply:
It's used in WordStorage either.